src/System.Reflection.DispatchProxy/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,46 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0003</DiagnosticId>
+    <Target>System.Reflection.DispatchProxy, Version=4.0.6.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51</Target>
+    <Left>lib/net461/System.Reflection.DispatchProxy.dll</Left>
+    <Right>lib/netstandard2.0/System.Reflection.DispatchProxy.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0003</DiagnosticId>
+    <Target>System.Reflection.DispatchProxy, Version=4.0.6.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51</Target>
+    <Left>lib/netcoreapp2.0/System.Reflection.DispatchProxy.dll</Left>
+    <Right>lib/netstandard2.0/System.Reflection.DispatchProxy.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0003</DiagnosticId>
+    <Target>System.Reflection.DispatchProxy, Version=4.0.6.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51</Target>
+    <Left>lib/netstandard2.0/System.Reflection.DispatchProxy.dll</Left>
+    <Right>lib/netstandard2.0/System.Reflection.DispatchProxy.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0003</DiagnosticId>
+    <Target>System.Reflection.DispatchProxy, Version=4.0.6.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51</Target>
+    <Left>ref/net461/System.Reflection.DispatchProxy.dll</Left>
```

Comment:
Ignore net461 as well. Already out of support.

Reply:
Putting all of these in a single PR: https://github.com/dotnet/maintenance-packages/pull/53